databases/percona/Portfile

```diff
@@ -21,29 +22,54 @@ if {$subport eq $name} {
 
     PortGroup           cmake 1.0
```

Comment:
Can you please switch to the `cmake 1.1` portgroup, remove `cmake.out_of_source yes`, and test it? This is a long standing issue. See https://trac.macports.org/ticket/47197. The ticket says it is fixed, but it did not get migrated to the cmake 1.1 portgroup.

Reply:
Thanks fixed.